packages/ui/src/models/visualization/base-visual-entity.ts

```diff
@@ -84,6 +84,8 @@ export interface IVisualizationNode<T extends IVisualizationNodeData = IVisualiz
 
   getNodeInteraction(): NodeInteraction;
 
+  addNodeInteraction(nodeInteraction: NodeInteraction): void;
```

Comment:
[question]: Since we can only set a single `nodeInteraction`, wouldn't be better to call it `setNodeInteraction` instead? 
```suggestion
  setNodeInteraction(nodeInteraction: NodeInteraction): void;
```